pkgs/desktops/xfce/core/gtk-xfce-engine.nix

```diff
@@ -1,17 +1,19 @@
-{ stdenv, fetchurl, pkgconfig, intltool, gtk }:
+{ stdenv, fetchurl, pkgconfig, intltool, gtk, withGtk3 ? false, gtk3 }:
```

Comment:
Xfce is intended as a light-weight desktop environment. Making it depend on two versions of GTK+ goes against that.